ssl/statem/statem_clnt.c

```diff
@@ -2674,6 +2675,32 @@ MSG_PROCESS_RETURN tls_process_new_session_ticket(SSL *s, PACKET *pkt)
 
     /* This is a standalone message in TLSv1.3, so there is no more to read */
     if (SSL_IS_TLS13(s)) {
+        const EVP_MD *md = ssl_handshake_md(s);
+        int hashleni = EVP_MD_size(md);
+        size_t hashlen;
+        const char nonce_label[] = "resumption";
```

Comment:
This can/should be `static` too. BTW, it occurred to me, is it actually so that it can't be declared unsigned char, so that one doesn't have to use explicit cast later on? Naturally applies even to second similar occurrence...

Reply:
Fixed this and the occurrences in extensions.c. I'm not actually sure why this wasn't declared unsigned. I seem to recall some problem with doing that, but can't now recreate it - so I changed this and all other occurrences to avoid unnecessary casts.

Reply:
It's `unsigned char *str = "resumption"` that would be problematic, but not `unsigned char str[] = "resumption"`.